pages/docs/administration/getting-started/helm-deployments.mdx

```diff
@@ -0,0 +1,35 @@
+---
+priority: 10200
+---
+
+import DocsWrapper from '@/src/docs/DocsWrapper'
+
+# Helm Deployments
+
+export default ({ children }) => <DocsWrapper>{children}</DocsWrapper>
+
+## Note
+
+Below deployment is configured for OpenShift. You will have to change route configuraiton to ingress for k8s.
+
+## Prerequisite
+
+- OpenShift CLI: https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz
```

Comment:
Use the Markdown format for links, `[name](http://example.com)`

Reply:
complete.